openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -158,7 +158,8 @@ class CreditApiTestBase(ModuleStoreTestCase):
 
     def setUp(self, **kwargs):
         super(CreditApiTestBase, self).setUp()
-        self.course_key = CourseKey.from_string("edX/DemoX/Demo_Course")
+        self.course = CourseFactory.create(org="edx", course="DemoX", run="Demo_Course")
+        self.course_key = self.course.id
```

Comment:
Is `self.course_key` used?  Don't see it in the PR diff.

Reply:
I think it's used in preexisting tests, since `self.course_key` was set in the old `setUp(...)` method as well. The substantive change here is constructing `self.course` in the setUp method instead of the individual test(s) that use(s) it explicitly, since now these tests all need an actual course exist in order to prevent the signals I created from causing failures when the grading code looks for a course and doesn't find one.